src/HttpHeaders.php

```diff
@@ -123,16 +123,27 @@ public function send()
         }
     }
 
+    /**
+     * @return array
+     */
+    public function getSentHeaders()
+    {
+        /** @link http://php.net/manual/ru/function.headers-list.php#120539 */
+        return (PHP_SAPI === 'cli' && \function_exists('xdebug_get_headers'))
```

Comment:
why `\ function_exists `, it will work without slash as well.

Reply:
It is a good practice to use the FQN on global functions cos the original class may be refactored and moved to non-root namespace.